src/main/java/net/torocraft/minecoprocessors/util/BookCreator.java

```diff
@@ -17,59 +20,44 @@
 
   private static final String PATH = "/assets/minecoprocessors/books/";
   private static final String PAGE_DELIMITER = "~~~";
+  public static final ItemStack manual = new ItemStack(Items.WRITTEN_BOOK);
 
-  public static ItemStack createBook(String name) {
-    ItemStack book = null;
-    try {
-      book = loadBook(name);
-    } catch (Exception e) {
-      e.printStackTrace();
-    }
-
-    if (book == null) {
-      book = new ItemStack(Items.BOOK);
-    }
-    return book;
-  }
-
-  private static ItemStack loadBook(String name) throws IOException {
-    ItemStack book = new ItemStack(Items.WRITTEN_BOOK);
+  static {
     String line;
```

Comment:
Pull this code out of the static block and into a `createBook(name)` method, then call it on line 23.  That way we get the static reference and a reusable method.